internal/common/node_module_info.bzl

```diff
@@ -12,22 +12,41 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""NodeModuleInfo provider and apsect to collect node_modules from deps.
+"""NodeModuleInfo & NodeModuleSources providers and apsect to collect node_modules from deps.
 """
 
 NodeModuleInfo = provider(
-    doc = "This provider contains information about npm dependencies installed with yarn_install and npm_install rules",
+    doc = "Provides information about npm dependencies installed with yarn_install and npm_install rules",
     fields = {
         "workspace": "The workspace name that the npm dependencies are provided from",
     },
 )
 
+NodeModuleSources = provider(
```

Comment:
add a note explaining the diff btween these providers

Reply:
done